apis/api-journeys-modern/src/schema/journeyEventsExportLog/journeyEventsExportLog.spec.ts

```
@@ -0,0 +1,111 @@
+import { Prisma } from '.prisma/api-journeys-modern-client'
+
+import { getClient } from '../../../test/client'
+import { prismaMock } from '../../../test/prismaMock'
+import { graphql } from '../../lib/graphql/subgraphGraphql'
+
+type Journey = Prisma.JourneyGetPayload<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)